tests/unit/dataactbroker/test_fabs_derivations.py

```diff
@@ -136,7 +135,7 @@ def test_funding_agency_name(database):
     assert obj['funding_agency_name'] is None
 
     # when funding_agency_code is provided
```

Comment:
This comment is no longer correct. This entire test is no longer needed actually because the test below it covers all those bases. I can't comment down there but `funding_agency_code ` and `funding_agency_name` should be tested down there too. It could probably be renamed to `test_funding_agency_derivations` or something